common/system/tasks/buzzer_heartbeat.cpp

```diff
@@ -1,25 +1,59 @@
-
 #include "buzzer_heartbeat.h"
 
-#include "hardware_manager.h"
-
 static uint32_t last_time = 0;
-static bool isOn;
+// static bool isOn;
 static int buzzerId_ = 0;
+BuzzerState state = BuzzerState::PYRO;
 
 void buzzerHeartbeat_setBuzzer(int buzzerId) { buzzerId_ = buzzerId; }
 
+void pyroSoundOff(bool pyroContCheck, int& pyroCounter);
```

Comment:
Why is this forward declared inside of a C++ file? If it's meant to be external, put it in the header; otherwise, make it static? IDK

Reply:
Essentially to mess with Sam